server/src/test/java/org/eclipse/openvsx/storage/AzureBlobStorageServiceTest.java

```diff
@@ -61,6 +63,16 @@ public void testGetLocation() {
 
     @TestConfiguration
     static class TestConfig {
+        @Bean
+        public MeterRegistry registry() {
+            return new SimpleMeterRegistry();
+        }
+
+        @Bean
+        public UrlConfigService urlConfigService() {
+            return new UrlConfigService();
+        }
+        
```

Comment:
Can't you add these two beans to the `@MockBean` list? These beans are not used in the test.